front-end/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:16-alpine
+
+WORKDIR /frontend
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 3000
+
+CMD ["npm", "start"]
```

```suggestion
CMD ["npm", "dev"]
```
Por que usar `"start"` em vez de `"dev"`? Isso não afetaria a experiência de desenvolvimento por não ter o hot reload, mesmo com os volumes montados no [compose](https://github.com/tryber/sd-016-b-project-delivery-app/pull/132/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3)?

---

O React não tem um script "dev" por padrão, o que ele tem é o "start" que é usado durante o desenvolvimento. No deploy o correto seria rodar comando "npm run build" e servir apenas o resultado desse comando usando alguma outra tecnologia como por exemplo o nginx... Vale lembrar que o proprio "start" do react já tem o hot-reload habilitado.

---

Faz todo sentido, você está certo!